src/app/globals.css

```diff
@@ -40,46 +40,20 @@ a {
   text-decoration: none;
 }
 
+
 @media (prefers-color-scheme: dark) {
   html {
     color-scheme: dark;
   }
 }
 
-.not-found-container {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  min-height: 100vh;
-  text-align: center;
-}
-
-.not-found-container h1 {
-  font-size: 5rem;
-  color: #B077FF;
+button{
+  font-size: 28px;
+  font-weight: 400;
+  line-height: 27.89px;
+  height: 48px;
+  width: 324px;
+  background: #0c0811;
+  font-family: Cascadia Mono;
+  color: #ffffff;
```

Comment:
these need to be in a seperate CSS file with the active or inactive components, see how other components are done with this folder structure like bottom-nav for example